math.aqua

```diff
@@ -0,0 +1,42 @@
+service Math("math"):
+    -- x + y
+    add(x: i64, y: i64) -> i64
+    -- x - y
+    sub(x: i64, y: i64) -> i64
+    -- x * y
+    mul(x: i64, y: i64) -> i64
+    -- floor(x * y)
+    fmul(x: f64, y: f64) -> i64
+    -- x / y
+    div(x: i64, y: i64) -> i64
+    -- division remainder, x % y
+    rem(x: i64, y: i64) -> i64
+    -- x^y
+    pow(x: i64, y: u32) -> i64
+    -- logarithm of y to base x
+    -- x – logarithm base
+    log(x: i64, y: i64) -> u32
+
+service Compare("cmp"):
+    -- x > y
+    gt(x: i64, y: i64) -> bool
```

Comment:
I think it's worth adding `gte` and `lte` in addition to `gt` and `lt` for more convenience